test/hotspot/jtreg/compiler/lib/template_framework/CodeFrame.java

```diff
@@ -32,19 +32,22 @@
  * The {@link CodeFrame} represents a frame (i.e. scope) of code, appending {@link Code} to the {@code 'codeList'}
  * as {@link Token}s are rendered, and adding names to the {@link NameSet}s with {@link Template#addStructuralName}/
  * {@link Template#addDataName}. {@link Hook}s can be added to a frame, which allows code to be inserted at that
```

Comment:
Since we have template and code frames it might help to be more precise here:
```suggestion
 * {@link Template#addDataName}. {@link Hook}s can be added to a code frame, which allows code to be inserted at that
```